components/Workouts/AlphabetWorkoutList.jsx

```diff
@@ -83,7 +84,13 @@ const AlphabetWorkoutList = (props) => {
         name={left.name}
         subtext={left.subtext}
         displayEllipses={left.displayEllipses}
-        deleteWorkout={() => dispatch(actions.workouts.deleteWorkout(left.id))}
+        deleteWorkout={() => {
+          if (selectedCycle.workouts.includes(left.id)) {
+            alert('This workout is included in the selected cycle. Please selected another cycle before deleting this workout.');
```

Comment:
tiny grammar - select instead of selected in the second sentence

Reply:
oops hahah good catch